google/cloud/functions/internal/compiler_info_test.cc

```diff
@@ -31,7 +34,10 @@ TEST(CompilerInfo, CompilerVersion) {
   auto const cv = CompilerVersion();
   EXPECT_NE(cv, "");
   // Look for something that looks vaguely like an X.Y version number.
-  EXPECT_THAT(cv, ::testing::ContainsRegex(R"([0-9]+.[0-9]+)"));
+  // Cannot use a regex because GTest does not have them with MSVC.
+  EXPECT_THAT(cv, Contains('.'));
+  EXPECT_TRUE(std::regex_match(cv, std::regex(R"(^[0-9]+\.[0-9]+.*)")))
```

Comment:
`std::regex_match()` doesn't need the `^` because it is always anchored.

Perhaps you would prefer `std::regex_search()`, keeping the leading `^`, but dropping the trailing `.*`.

[All that said, the old code wasn't anchored at all.]

Reply:
Done (unanchored).